rh/views.py

```diff
@@ -0,0 +1,3 @@
+# from django.shortcuts import render
+
+# Create your views here.
```

Comment:
Err can you get rid of this file too?

Reply:
I guess `manage.py startapp` is more trouble than it's worth since we just end up deleting most of the files it makes, lol.